versions/1.1/SPEC.md

```diff
@@ -0,0 +1,5775 @@
+* `wf.foobar.input` - References the `File` input of the call to `other.foobar`
+* `wf.other_workflow` - References the first call to subworkflow `other.other_workflow`
+* `wf.other_workflow.bool` - References the `Boolean` input of the first call to subworkflow `other.other_workflow`
+* `wf.other_workflow.foobar.results` - References the `File` output of the call to `foobar` inside the first call to subworkflow `other.other_workflow`
+* `wf.other_workflow.foobar.input` - References the `File` input of the call to `foobar` inside the first call to subworkflow `other.other_workflow`
+* `wf.other_workflow2` - References the second call to subworkflow `other.other_workflow` (aliased as other_workflow2)
+* `wf.other_workflow2.bool` - References the `Boolean` input of the second call to subworkflow `other.other_workflow`
+* `wf.other_workflow2.foobar.results` - References the `File` output of the call to `foobar` inside the second call to subworkflow `other.other_workflow`
+* `wf.other_workflow2.foobar.input` - References the `File` input of the call to `foobar` inside the second call to subworkflow `other.other_workflow`
+* `wf.arr` - References the `Array[String]` declaration on the workflow
+* `wf.scattered_test` - References the scattered version of `call test`
+* `wf.scattered_test.my_var` - References an `Array[String]` for each element used as `my_var` when running the scattered version of `call test`.
+* `wf.scattered_test.results` - References an `Array[File]` which are the accumulated results from scattering `call test`
+* `wf.scattered_test.1.results` - References an `File` from the second invocation (0-indexed) of `call test` within the scatter block.  This particular invocation used value "b" for `my_var`
+
+A namespaced identifier has the same syntax as a fully-qualified name. It is interpreted as the left-hand side being the name of a namespace and the right-hand side being the name of a workflow, task, or sub-namespace within that namespace.
+
+Consider this workflow:
+
+```wdl
+import "other.wdl" as ns
+
+workflow wf {
+  call ns.ns2.task
+}
+```
+
+Here, `ns.ns2.task` is a namespace identifier (see the [Call Statement](#call-statement) section for more details). Namespace identifiers, like fully-qualified names are left-associative, which means `ns.ns2.task` is interpreted as `((ns.ns2).task)`, which means `ns.ns2` has to resolve to a namespace so that `.task` can be applied. If `ns2` is instead a task definition within `ns`, then the namespaced identifier `ns.ns2.task` is invalid.
+
+### Declarations
+
+```txt
+$declaration = $type $identifier ('=' $expression)?
+```
+
+A declaration reserves a name that can be referenced anywhere in the [scope](#appendix-b-wdl-namespaces-and-scopes) where it is declared. A declaration has a type, a name, and an optional initialization. Each declaration must be unique within its scope, and may not collide with a reserved WDL keyword (e.g. `workflow`, or `input`).
```

Comment:
maybe add a link here to the reserved keyword section?

Reply:
done